barretenberg/cpp/src/barretenberg/hypernova/hypernova_decider_verifier.hpp

```diff
@@ -3,7 +3,10 @@
 // external_1:  { status: not started, auditors: [], date: YYYY-MM-DD }
 // external_2:  { status: not started, auditors: [], date: YYYY-MM-DD }
 // =====================
-
+//
```

Comment:
Any reason not to make these and other similar comments doxy style on the class itself?